tests/integration/test_sns.py

```diff
@@ -550,16 +552,17 @@ def test_publish_sms(self, sns_client):
         assert "MessageId" in response
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-    @pytest.mark.only_localstack
-    def test_publish_non_existent_target(self, sns_client):
+    @pytest.mark.aws_validated
+    def test_publish_non_existent_target(self, sns_client, sns_create_topic, snapshot):
         # todo: fix test, the client id in the ARN is wrong so can't test against AWS
```

Comment:
Is this todo addressed now? was the "client id" the account id?

Reply:
Yes, I forgot to remove the todo! Thanks a lot 🙏